bootcamp/tutorials/integration/function_calling.ipynb

```diff
@@ -0,0 +1,430 @@
+{
```

Comment:
<p>ditto, some quick explaination rather than diving into code immediately.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/milvus-io/bootcamp/pull/1391/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>